### DISCUSSION TITLE
No files in repos?

### DISCUSSION THREAD

**Discussion by scorputer** [community/community#54463] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've been offline a couple of weeks, setting up a new computer.  I logged in to github and do not see any of the code I checked in.  It's as if the files were all removed.    I'm certain I must be overlooking something simple as I doubt all my files disappeared (?)  Has anyone here run into this before?

**Comment by airtower-luna** [comment]
> I logged in to github and do not see any of the code I checked in. It's as if the files were all removed.

*Where* do you not see the code? If it's not on the new computer, that's natural: You'll need to actually clone the repositories you're working with. If it's not visible on GitHub that's be worrisome. :sweat_smile:

**Reply by scorputer** [reply]
LOL! The files didn't teleport to the new computer - but I wasn't expecting that.  I'm saying that when I log onto github, I see no files in my (4) repos.  It's like the repo folders were made but there's no evidence a file was checked in.  FWIW, I see the same when I try looking from my phone as well.  I know I did have files checked in.  I'm fairly sure I did not click any big button that said "wipe my repos".  Local clones of my repos also lack files.

**Reply by spenserblack** [reply, marked as answer]
I guess they're private repos, since [this](https://github.com/scorputer/scorputer) is all that I found.
So, just to confirm, you did log in to GitHub see your files **in the web client** *before* changing computers?

Sorry for asking a dumb question, but you *did* call `git push`, `gh repo sync`, or some other equivalent action, right?

**Reply by scorputer** [reply]
It's the right question.  As there are no receipts of files on github, I think that's probably what didn't happen.  I was working off a local copy of the repo, the files were committed locally and the committed files were not pushed :-/  Mystery solved, note to self made.  Thanks!
